swagger/v1/openapi.json

```diff
@@ -76,46 +105,48 @@
               "application/vnd.api+json": {
                 "data": [
                   {
-                    "id": "aa67c98c-d81f-5a9c-b0bc-26caa0051aea",
+                    "id": "9d9cb70a-024a-4152-9a26-1df1f400f392",
```

Comment:
`git add -i swagger` is your friend :) There's a lot of good example responses here from before FactoryBot that are now either empty or show less relationships. I'd like to go back through the specs and ensure we have similar data there as before. Until then, though, there's no need to overwrite parts of the openapi.json that aren't affected by this sorting change.